### PR TITLE
Added benchmark for early removal of unused raster cache entris mecha…

### PR DESCRIPTION
**Pull request by ColdPaleLight**
benchmark for https://github.com/flutter/engine/pull/34627

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [ ] I signed the [CLA].
- [ ] I listed at least one issue that this PR fixes in the description above.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [ ] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

File: dev/benchmarks/macrobenchmarks/lib/src/color_filter_with_unstable_child.dart [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+class _ColorFilterWithUnstableChildPageState extends State<ColorFilterWithUnstableChildPage> with SingleTickerProviderStateMixin {
+  late Animation<double> _offsetY;
+  late AnimationController _controller;
+
+  @override
+  void initState() {
+    super.initState();
+
+    _controller = AnimationController(vsync: this, duration: const Duration(seconds: 2));
+    // Animations are typically implemented using the AnimatedBuilder widget.
```
Comment (flar) [inline dev/benchmarks/macrobenchmarks/lib/src/color_filter_with_unstable_child.dart:23]: This benchmark has no history to maintain. It looks like this comment and implementation were cut/pasted from some other benchmark that had to deal with consistency, but that doesn't apply here. Shouldn't it use the more recommended mechanism in this case?

**Review by flar — CHANGES_REQUESTED** [review]

File: dev/benchmarks/macrobenchmarks/lib/src/color_filter_with_unstable_child.dart (hunk 1 above)
Comment (ColdPaleLight) [reply dev/benchmarks/macrobenchmarks/lib/src/color_filter_with_unstable_child.dart]: Done, using AnimatedBuilder now.

**Review by flar — APPROVED** [review]
LGTM

**Comment by flar** [comment]
I went ahead and added the autosubmit label so this can get into the tree to start gathering baseline data.

**Pull request merged**